src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is working right now!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working!");
```

Comment:
Similarly, the stop message 'Excavator stopped working!' could be more descriptive, such as 'Excavator has finished digging and is now idle.' to provide clearer information about the machine's state.